doc/publications.rst

```diff
@@ -19,12 +19,18 @@ Refereed Publications
 
       * - Kukulies, J., Chen, D., & Curio, J. (2021). The role of mesoscale convective systems in precipitation in the Tibetan Plateau region. *Journal of Geophysical Research: Atmospheres*, 126(23), e2021JD035279. doi:10.1029/2021JD035279.
 
+      * - Kukulies, J., Lai, H. W., Curio, J., Feng, Z., Lin, C., Li, P., ... & Chen, D. (2023). Mesoscale convective systems in the Third pole region: Characteristics, mechanisms and impact on precipitation. *Frontiers in Earth Science*, 11, 1143380.
```

Comment:
Same comment here about author list. I don't have a preference, but perhaps showing full author lists (within reason) would be nice?